packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/AppContext.kt

```diff
@@ -185,6 +199,11 @@ class AppContext(
   }
 
   fun onHostResume() {
+    activityResultsManager.onHostResume(
+      requireNotNull(currentActivity) {
+        "Current Activity is not available at this moment"
+      }
```

Comment:
This would actually never crash, because whenever any of lifecycle method is called (except for `OnCreate`) the current `Activity`  must be available, because it is actually propagating these events 😉 
I went with `requireNotNull` to get rid of `nullability` (I could have gone with `!!`, but we don't want that, do we? 👀).
I can go with some other message, like: `"This would never happen, but if it does, something is broken very much"`

Reply:
I think you can leave it as it is.